docs/source/command_line.rst

```diff
@@ -129,17 +129,17 @@ imports.
 ``--python-executable EXECUTABLE``
     This flag will have mypy collect type information from :pep:`561`
     compliant packages installed for the Python executable ``EXECUTABLE``.
-    If not provided, mypy will use :pep:`561` compliant packages installed for
```

Comment:
There is a link to PEP 561 already one line above, the other ones were too much blue in one paragraph...?

Reply:
OK.